protocol/contracts/lpp/src/state/total.rs

```diff
@@ -102,19 +110,25 @@ where
         loan_principal_payment: Coin<Lpn>,
         loan_interest_rate: Percent,
     ) -> &Self {
-        self.total_interest_due = self.total_interest_due_by_now(&ctime) - loan_interest_payment;
+        // Due to rounding in the calculations, there is a possibility of overflow in the arithmetic operations.
```

Comment:
the comment should shed light on:
rounding of what exact calculation?
why it's safe to saturate to 0?